components/card/card.css

```diff
@@ -0,0 +1,163 @@
+.card-vertical {
+    box-shadow: 0 1rem 2rem 0 rgba(0, 0, 0, 0.2);
+    max-width: 25rem;
+    height:45rem;
+    margin: auto;
+    text-align: center;
+    font-family: arial;
+    border-radius: 1rem;
+    display:flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+
+  .card-v-header{
+      height:3%;
+      margin:0.3rem 0.2rem;
```

Comment:
have margin and padding in multiples of 4, and you can use px here as rem wont have much effects in terms of responsiveness

Reply:
noted. thanks Partha for the review

Reply:
Hi @partha8 can you kindly approve the PR if its good to be merged?